playground/infrastructure/api/v1/api_pb2_grpc.py

```diff
@@ -53,19 +53,24 @@ def __init__(self, channel):
             request_serializer=api__pb2.GetCompileOutputRequest.SerializeToString,
             response_deserializer=api__pb2.GetCompileOutputResponse.FromString,
         )
-        self.GetListOfExamples = channel.unary_unary(
```

Comment:
Why is it here? this changes already in master

Reply:
`api.v1.api.proto` file doesn't contain `GetListOfExamples` method. All these changes are the result of regenerating proto files using:
```
python -m grpc_tools.protoc --python_out=infrastructure/api/v1 --grpc_python_out=infrastructure/api/v1 --proto_path=api/v1 api.proto
```

Reply:
ok